specification/Styling/README.adoc

```diff
@@ -0,0 +1,1496 @@
+
+[source,json]
+----
+{
+    "color" : "color('red')",
+    "pointSize" : "${Temperature} * 0.5"
+}
+----
+
+Implementations may clamp the evaluated `pointSize` to the system's supported point size range. For example, WebGL renderers may query `ALIASED_POINT_SIZE_RANGE` to get the system limits when rendering with `POINTS`. A `pointSize` of `1.0` must be supported.
+
+Point Cloud styles may also reference semantics from the xref:{url-specification-tileformats-pointcloud}README.adoc#tileformats-pointcloud-feature-table[Feature Table] including position, color, and normal to allow for more flexible styling of the source data.
+
+* `+${POSITION}+` is a `vec3` storing the xyz Cartesian coordinates of the point before the `RTC_CENTER` and tile transform are applied. When the positions are quantized, `+${POSITION}+` refers to the position after the `QUANTIZED_VOLUME_SCALE` is applied, but before `QUANTIZED_VOLUME_OFFSET` is applied.
+* `+${POSITION_ABSOLUTE}+` is a `vec3` storing the xyz Cartesian coordinates of the point after the `RTC_CENTER` and tile transform are applied. When the positions are quantized, `+${POSITION_ABSOLUTE}+` refers to the position after the `QUANTIZED_VOLUME_SCALE`, `QUANTIZED_VOLUME_OFFSET`, and tile transform are applied.
+* `+${COLOR}+` evaluates to a `Color` storing the rgba color of the point. When the Feature Table's color semantic is `RGB` or `RGB565`, `+${COLOR}.alpha+` is `1.0`. If no color semantic is defined, `+${COLOR}+` evaluates to the application-specific default color.
+* `+${NORMAL}+` is a `vec3` storing the normal, in Cartesian coordinates, of the point before the tile transform is applied. When normals are oct-encoded, `+${NORMAL}+` refers to the decoded normal. If no normal semantic is defined in the Feature Table, `+${NORMAL}+` evaluates to `undefined`.
+
+For example:
+
+[source,json]
+----
+{
+    "color" : "${COLOR} * color('red')'",
+    "show" : "${POSITION}.x > 0.5",
+    "pointSize" : "${NORMAL}.x > 0 ? 2 : 1"
+}
+----
+
+[NOTE]
+.Implementation Note
+====
+Point cloud styling engines may often use a shader (GLSL) implementation, however some features of the expression language are not possible in pure a GLSL implementation. Some of these features include:
+
+* Evaluation of `isNaN` and `isFinite` (GLSL 2.0+ supports `isnan` and `isinf` for these functions respectively)
+* The types `null` and `undefined`
+* Strings, including accessing object properties (`color()['r']`) and batch table values
+* Regular expressions
+* Arrays of lengths other than 2, 3, or 4
+* Mismatched type comparisons (e.g. `1.0 === false`)
+* Array index out of bounds
+====
+
+[#styling-file-extension-and-mime-type]
+== File extension and MIME type
+
+Tileset styles use the `.json` extension and the `application/json` mime type.
```

Comment:
Not directly related to this PR but I noticed that some pages still use "MIME type" instead of "Media Type"

Reply:
Added this as a note at the "TODO list" comment below

Reply:
Created https://github.com/CesiumGS/3d-tiles/pull/690 and https://github.com/CesiumGS/3d-tiles/pull/691 - when the first one is merged, I'll update this (`asciidoc-conversion`) branch with that state.